test/unit/test_inference.py

```diff
@@ -17,6 +17,31 @@
 import sockeye.inference
 
 
+_BOS = 0
```

Comment:
Would it make sense to put this into constants or make it equivalent to whats actually happening? its already implicitly there in VOCAB_SYMBOLS: EOS is index 3 in VOCAB_SYMBOLS so EOS always gets id 3

Reply:
I left it in the test, as the function should be independent of the concrete values of BOS and EOS defined elsewhere.

Reply:
Ok